pkg/cmd/admin/policy/policy.go

```diff
@@ -165,6 +166,21 @@ func (a LocalRoleBindingAccessor) GetExistingRoleBindingNames() (*sets.String, e
 	return ret, nil
 }
 
+func (a LocalRoleBindingAccessor) GetRoleBinding(roleNamespace, roleBindingName string) (*authorizationapi.RoleBinding, error) {
+	roleBindings, err := a.Client.PolicyBindings(a.BindingNamespace).Get(authorizationapi.GetPolicyBindingName(roleNamespace), metav1.GetOptions{})
```

Comment:
why is this not just `a.Client.RoleBindings(namespace).Get(name, metav1.GetOptions{})`?